README.rst

```diff
@@ -1,8 +1,8 @@
 django-bitfield
 ---------------
 
-.. image:: https://api.travis-ci.org/disqus/django-bitfield.png?branch=master
-    :target: https://travis-ci.org/disqus/django-bitfield
+.. image:: https://api.travis-ci.org/znotdead/django-bitfield.png?branch=master
```

Comment:
I think they gave up this project and anyway they won't approve while it's doesn't support old Django 1.4.